src/components/sendV2/transactionStatus/statusMessages.js

```diff
@@ -0,0 +1,26 @@
+/* istanbul ignore file */
+import svg from '../../../utils/svgIcons';
+
+export default {
+  success: {
+    headerIcon: svg.transactionSuccess,
+    bodyText: {
+      title: 'Transaction submitted',
+      paragraph: 'You will find it in My Transactions in a matter of minutes',
+    },
+  },
+  error: {
+    headerIcon: svg.transactionError,
+    bodyText: {
+      title: 'Transaction failed',
+      paragraph: 'Oops, looks like something went wrong. Please try again.',
+    },
+  },
+  hw: {
+    headerIcon: svg.transactionError,
+    bodyText: {
+      title: 'Transaction aborted on device',
+      paragraph: 'You have cancelled the transaction on your harware wallet.',
+    },
+  },
+};
```

Comment:
Should be on the component, so it could be translated with the `t` function, otherwise it will only be interpolated and not translated.

Reply:
done